book/_config.yml

```diff
@@ -14,12 +14,11 @@ parse:
     dates: "October ? to ?, 2021"
     github_org_url: "https://github.com/uwhackweek"
     book_repo: "jupyterbook-template"
-    website_url: "https://uwhackweek.github.io/jupyterbook-template"
+    website_url: "https://book-template.hackweek.io"
     jupyterhub_url: "https://INSERT_JUPYTERHUB_URL"
     slack_workspace_url: "https://INSERT_SLACK_URL"
-    docker_image: "https://INSERT_DOCKER_IMAGE_URL"
```

Comment:
@scottyhq - Since we are no longer providing a docker image, I am wondering if we should also remove the associated GitHub workflows:
https://github.com/uwhackweek/jupyterbook-template/tree/main/.github/workflows

Reply:
Logged here #132